test/e2e/perftype/perftype.go

```diff
@@ -18,6 +18,11 @@ package perftype
 
 // TODO(random-liu): Replace this with prometheus' data model.
 
+// CurrentKubeletPerfMetricsVersion is the current kubelet performance metrics
+// version. This is used by mutiple perf related data structures. We should
+// bump up the version each time we make an incompatible change to the metrics.
+const CurrentKubeletPerfMetricsVersion = "v2"
```

Comment:
For internal use, I think it's fine to put it in perf_util.
`e2e_node` uses the same test framework with e2e.